tools/syn2mas/package.json

```diff
@@ -8,23 +8,27 @@
     "build": "tsc",
     "dev": "ts-node --esm src/index.ts",
     "lint": "npm run lint:types && npm run lint:style",
-    "lint:style": "eslint src",
-    "lint:types": "tsc --noEmit --skipLibCheck",
+    "lint:style": "eslint . .eslintrc.cjs",
+    "lint:types": "tsc --noEmit",
     "start": "node dist/index.js"
   },
   "devDependencies": {
+    "@tsconfig/node18": "^18.2.2",
+    "@tsconfig/strictest": "^2.0.2",
     "@types/command-line-args": "^5.2.1",
     "@types/node": "^18.18.4",
     "@typescript-eslint/eslint-plugin": "^6.7.5",
     "@typescript-eslint/parser": "^6.7.5",
     "eslint": "^8.51.0",
     "eslint-config-google": "^0.14.0",
     "eslint-config-prettier": "^9.0.0",
-    "eslint-plugin-import": "^2.28.1",
+    "eslint-import-resolver-typescript": "^3.6.1",
+    "eslint-plugin-import": "npm:eslint-plugin-i@^2.28.1",
     "eslint-plugin-matrix-org": "^1.2.1",
+    "eslint-plugin-prettier": "^5.0.0",
     "eslint-plugin-unicorn": "^48.0.1",
     "prettier": "^3.0.3",
-    "ts-node": "^10.9.1",
+    "ts-node": "github:TypeStrong/ts-node#main",
```

Comment:
There is a bug in the npm released version which makes it crash when there is an array of `extends` in the tsconfig